pandas/_typing.py

```diff
@@ -507,3 +507,12 @@ def closed(self) -> bool:
 
 # Offsets
 OffsetCalendar = Union[np.busdaycalendar, "AbstractHolidayCalendar"]
+
+# read_csv: usecols
+UsecolsArgType = Union[
+    SequenceNotStr[Hashable],
+    Sequence[int],
```

Comment:
needed for `range`: `range` inherits from `Sequence` but changes some of the signatures so it doesn't fit the `SequenceNotStr` protocol

Reply:
Why not just be explicit with `range` ?

Reply:
changed it to range